src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -192,10 +194,8 @@ private void dropItems(BlockBreakEvent e, List<ItemStack> drops) {
      * Sensitive {@link Block Blocks} are pressure plates or saplings, which should be broken
      * when the block beneath is broken as well.
      *
-     * @param p
-     *            The {@link Player} who broke this {@link Block}
-     * @param b
-     *            The {@link Block} that was broken
+     * @param p The {@link Player} who broke this {@link Block}
+     * @param b The {@link Block} that was broken
```

Comment:
Looks like a reformat or auto import has changed the order/formatting here - best revert back to how it was.

Reply:
fixed